src/pages/home/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
I'm neither a React or CSS expert but Brian Burdan suggested we look at jss for React apps.

Reply:
I just changed this